### PR TITLE
Apollo vanjee lidar driver

### PR DESCRIPTION
**Pull request by lvvmao**
1、添加万集激光雷达驱动；
2、已经进行过功能测试；

### CODE REVIEW

File: modules/drivers/lidar/vanjeelidar/BUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        "src/vanjeelidar_component.h",
+    ],
+    deps = [
+        "//modules/common_msgs/sensor_msgs:pointcloud_cc_proto",
+        "//modules/drivers/lidar/vanjeelidar/proto:vanjeelidar_config_proto",
+        "//modules/drivers/lidar/vanjeelidar/proto:vanjeelidar_proto",
+        "//modules/drivers/lidar/common:lidar_common",
+    ],
+    linkopts = [
+        "-lpcap",
```
Comment (daohu527) [inline modules/drivers/lidar/vanjeelidar/BUILD:24]: Is pcap required?

File: modules/drivers/lidar/vanjeelidar/src/vanjeelidar_component.cpp [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "modules/drivers/lidar/vanjeelidar/src/vanjeelidar_component.h"
+
+#include <string>
+
+#include <vanjee_driver/driver/input/input.hpp>
```
Comment (daohu527) [inline modules/drivers/lidar/vanjeelidar/src/vanjeelidar_component.cpp:21]: Where are the files? If have source code, you can add in the code

File: modules/drivers/lidar/vanjeelidar/BUILD (hunk 1 above)
Comment (lvvmao) [reply modules/drivers/lidar/vanjeelidar/BUILD]: It is  not necessary,we  had delete “-lpcap”.

File: modules/drivers/lidar/vanjeelidar/src/vanjeelidar_component.cpp (hunk 2 above)
Comment (lvvmao) [reply modules/drivers/lidar/vanjeelidar/src/vanjeelidar_component.cpp:19]: we can offer the source code,and pack them as a deb.

**Review by daohu527 — APPROVED** [review]
LGTM

**Comment by xinxindream** [comment]
- fatal error: vanjee_driver/api/lidar_driver.hpp: No such file or directory
- bash apollo.sh build meets an error
```
ERROR: /apollo/modules/drivers/lidar/vanjeelidar/BUILD:8:17: Compiling modules/drivers/lidar/vanjeelidar/src/vanjeelidar_component.cpp failed: (Exit 1): crosstool_wrapper_driver_is_not_gcc failed: error executing command external/local_config_cuda/crosstool/clang/bin/crosstool_wrapper_driver_is_not_gcc -MD -MF ... (remaining 116 arguments skipped)
In file included from modules/drivers/lidar/vanjeelidar/src/vanjeelidar_component.cpp:18:0:
./modules/drivers/lidar/vanjeelidar/src/vanjeelidar_component.h:20:10: fatal error: vanjee_driver/api/lidar_driver.hpp: No such file or directory
 #include <vanjee_driver/api/lidar_driver.hpp>
```
maybe there are some files missing?

**Comment by daohu527** [comment]
@xinxindream yes, revert the commit and will update later

**Pull request merged**